package.json

```diff
@@ -24,11 +24,9 @@
     "format-fix": "pnpm format --write --list-different",
     "lint-prune": "! ts-prune | grep -v \"used in module\" | grep -v rpc | grep -v observable | grep -v http",
     "clean": "find . -name node_modules -o -name .turbo -o -name .next -o -name dist -o -name __generated__ -type d -prune | xargs rm -rf",
-    "codegen:tests": "turbo codegen-tests",
-    "codegen:entrypoints": "turbo codegen-entrypoints",
-    "codegen": "run-p codegen:*",
+    "codegen-tsconfigs": "tsx scripts/tsconfigs.ts",
     "prepublish": "pnpm build",
-    "postinstall": "pnpm codegen",
+    "postinstall": "turbo codegen-tests",
```

Comment:
our `next-sse-chat` deployment got issues because there were scripts in parallel that touched the same file

now we just use `codegen-tests` which has the rest as deps


shower thought: we could just have one codgen-script in our root that did all of this as it is so fast to do these